DatadogTrace/Tests/ContextMessageReceiverTests.swift

```diff
@@ -67,6 +68,56 @@ class ContextMessageReceiverTests: XCTestCase {
         XCTAssertNil(receiver.context.rum?["_dd.action.id"])
     }
 
+    func testItReceivesNilRUMContext() throws {
+        // Given
+        let receiver = ContextMessageReceiver(bundleWithRumEnabled: true)
+        let core = PassthroughCoreMock(
+            context: .mockWith(
+                baggages: [
+                    "rum": .init([
+                        "application.id": "app-id",
+                        "session.id": "session-id",
+                        "view.id": "view-id",
+                        "user_action.id": "action-id"
+                    ])
+                ]
+            ),
+            messageReceiver: receiver
+        )
+
+        XCTAssertEqual(receiver.context.rum?["_dd.application.id"], "app-id")
+        XCTAssertEqual(receiver.context.rum?["_dd.session.id"], "session-id")
+        XCTAssertEqual(receiver.context.rum?["_dd.view.id"], "view-id")
+        XCTAssertEqual(receiver.context.rum?["_dd.action.id"], "action-id")
+
+        // When
+        core.set(baggage: nil, forKey: "rum")
+
+        // Then
+        XCTAssertNil(receiver.context.rum)
+    }
```

Comment:
Similar problem to other `MessageReceiver` tests. We test the receiver's integration with **mocked** core, not the receiver itself. The same in `testItReceivesMalformedRUMContext()`.